user_guide_src/source/tutorial/index.rst

```diff
@@ -59,7 +59,7 @@ From your command line type the following:
 
 ::
 
-    ➜ composer create-project codeigniter4/appstarter ci-blog -s rc
+    composer create-project codeigniter4/appstarter ci-blog -s rc
```

Comment:
Do we want to drop `-s rc` now that we're a legit stable release? I think so, and then we have the `./builds` commands for switching to source as needed.

Reply:
Oh, so that's what it's for. Will remove it. :-)